platform/mwm_version.cpp

```diff
@@ -13,6 +13,13 @@
 
 namespace version
 {
+int64_t constexpr kASingleMwmVersionForTesting1 = 991215;
```

Comment:
Moreover, the type of these variables is "int64_t constexpr" whereas in the header file they're defined as "int64_t const". Not sure that this is correct in accordance with the C++ standard.